engines/postgres.py

```diff
@@ -130,5 +131,9 @@ def get_connection(self):
                              user=self.opts["user"],
                              password=self.opts["password"],
                              database=self.opts["database"])
-        conn.set_client_encoding('Latin1')
+
+        encoding = self.script.encoding
+        encoding_lookup = {'ISO-8859-1': 'LATIN1', 'utf-8': 'UTF8'}
```

Comment:
Sure I would make those changes. I was a little confused about the encoding lookup table as to which encoding will map to which one in postgres and mysql database. Did I made them right?